src/main/webapp/user-page.html

```diff
@@ -16,31 +16,60 @@
 
 <!DOCTYPE html>
 <html>
-  <head>
+<head>
     <title>User Page</title>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
     <script src="/js/user-page-loader.js"></script>
-  </head>
-  <body onload="buildUI();">
-    <nav>
-      <ul id="navigation">
+</head>
+
+<body onload="buildUI();">
+<nav>
+    <ul id="navigation">
         <li><a href="/">Home</a></li>
-      </ul>
-    </nav>
-    <h1 id="page-title">User Page</h1>
-
-    <form id="message-form" action="/messages" method="POST" class="hidden">
-      Enter a new message:
-      <br/>
-      <textarea name="text" id="message-input"></textarea>
-      <br/>
-      <input type="submit" value="Submit">
+    </ul>
+</nav>
+<h1 id="page-title">User Page</h1>
+
+<div class="userForm">
+    <h2>
+        What tea did you drink today?
```

Comment:
Given that the interpretation of "today" is dependent on the user's timezone, this form needs to have some way to propagate the date and time-zone of the user back to the server. If submitting this form in JavaScript, you could do this trivially (by retrieving that information and passing it along in the request). If you are using the built-in form action, and `<button type="submit">`, then you might be able to accomplish this by injecting an `<input>` with `type="hidden"` , where you have precomputed the date/timezone information and stored it in advance of the user hitting the submit button.